Makefile

```diff
@@ -82,6 +82,9 @@ pylint:
 pytype:
 	pytype
 
+mypy: clean
+	$(PYTHON) -m mypy --show-error-codes $(NAME)
```

Comment:
Not sure we need to call `python` here. Call `mypy` directly, like `pytype` above.